docs/Concepts/Authorization.md

```diff
@@ -41,9 +41,11 @@ QKM currently has the following resources:
 | Secret             | A key-value element stored in a secure vault system.                |
 | Key                | A cryptographic key.                                                |
 | Ethereum account   | A cryptographic key allowing interaction with the Ethereum network. |
-| [Store](Stores.md) | A set of secrets, keys, or Ethereum accounts.                       |
+| [Vault](Stores.md#Vault) | Vault Client connector use to persist resources remotely.                       |
```

Comment:
Do we want a separate page for this ? Or we put it in the Stores.md page?

Reply:
updated